src/indieweb_utils/indieauth/happ.py

```diff
@@ -14,26 +14,35 @@ class ApplicationInfo:
     summary: str
 
 
-def get_h_app_item(
-    web_page: str,
-    client_id: str,
-    redirect_uri: str,
-) -> ApplicationInfo:
+def get_h_app_item(web_page: str, client_id: str) -> ApplicationInfo:
     """
     Get the h-app item from the web page.
 
     :param web_page: The web page to parse.
     :type web_page: str
     :param client_id: The client id of your application.
     :type client_id: str
-    :param redirect_uri: The redirect uri specified by the client.
-    :type redirect_uri: str
     :return: The h-app item.
     :rtype: ApplicationInfo
+
+    Example:
+
+    .. code-block:: python
+
+        import indieweb_utils
+
+        app_url = "https://quill.p3k.io/"
+        client_id = "https://quill.p3k.io/"
+
+        h_app_item = indieweb_utils.get_h_app_item(
+            app_url, client_id
+        )
+
+        print(h_app_item.name) # Quill
     """
 
-    redirect_uri_domain = parse_url(redirect_uri).netloc
-    redirect_uri_scheme = parse_url(redirect_uri).scheme
+    client_id_domain = parse_url(client_id).netloc
+    client_id_scheme = parse_url(client_id).scheme
```

Comment:
We're parsing the client_id twice when we could do it once.
```suggestion
    parsed_client_id = parse_url(client_id)
    client_id_domain = parsed_client_id.netloc
    client_id_scheme = parsed_client_id.scheme
```